src/app/cli/src/init/coda_run.ml

```diff
@@ -177,7 +177,8 @@ let make_report exn_str ~conf_dir ~top_logger coda_ref =
 let setup_local_server ?(client_whitelist = []) ?rest_server_port
     ?(insecure_rest_server = false) coda =
   let client_whitelist =
-    Unix.Inet_addr.Set.of_list (Unix.Inet_addr.localhost :: client_whitelist)
+    ref
```

Comment:
This variable might be problematic when migrating the commands to GraphQL and putting them inside Coda_graphql. You can probably put this variable inside Coda_commands as a global_variable or within Coda_lib.